skimage/feature/util.py

```diff
@@ -107,10 +107,10 @@ def plot_matches(ax, image1, image2, keypoints1, keypoints2, matches,
         image = np.concatenate([image1, image2], axis=0)
         offset[1] = 0
     else:
-        mesg = ("plot_matches accepts either 'horizontal' or 'vertical' for "
-                "alignment, but '{}' was given. See "
+        mesg = (f"plot_matches accepts either 'horizontal' or 'vertical' for "
+                "alignment, but '{alignment}' was given. See "
                 "https://scikit-image.org/docs/dev/api/skimage.feature.html#skimage.feature.plot_matches "  # noqa
-                "for details.").format(alignment)
+                "for details.")
```

Comment:
To be consistent with the style used in other recent f-string PRs, both this line immediately above should also be f-strings even though it is not technically necessary.

Reply:
Oh ok. Did you make all the necessary changes or is there something I should change?

Reply:
```suggestion
                f"for details.")
```

Reply:
I added suggestions for the last two lines just now. GitHub didn't let me select the block all as one due to the unchanged line in-between.